src/plugins/experimental-discord/createGraph.js

```diff
@@ -212,18 +219,28 @@ export function createGraph(
       const reactions = repo.reactions(channel.id, message.id);
       for (const reaction of reactions) {
         const emojiRef = Model.emojiToRef(reaction.emoji);
-        const reactionWeight = emojiWeights[emojiRef];
+        let reactionWeight = emojiWeights[emojiRef];
+        if (reactionWeight === null || reactionWeight === undefined) {
```

Comment:
`reactionWeight === null || reactionWeight === undefined` is equivalent to `reactionWeight == null`. It's the one good time to use `==` rather than `===` in js.

Reply:
Do we really want to use the soft equality like this? I'll change it though.

Reply:
Yeah, we use soft equality this way throughout the codebase. `git grep "== null" | wc -l` has 160 hits